cli/azd/cmd/auth_login.go

```
@@ -311,6 +311,14 @@ func (la *loginAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 		} else {
 			res.Status = contracts.LoginStatusSuccess
 			res.ExpiresOn = &token.ExpiresOn
+
+			// Get user account information for both JSON and non-JSON output
+			details, detailsErr := la.authManager.LogInDetails(ctx)
+			if detailsErr == nil {
+				res.Account = &details.Account
+				loginType := string(details.LoginType)
+				res.LoginType = &loginType
+			}
```

@weikanglim how u feel about ignoring the error here? 
Should we at least log.printf when error != nil?

---

@JeffreyCA can we use the copilot instructions to never ignore errors?

---

Yes I think we can directly update copilot-instructions.md in this PR too if we want